presto-accumulo/src/main/java/io/prestosql/plugin/accumulo/AccumuloSplitManager.java

```diff
@@ -52,7 +53,7 @@ public AccumuloSplitManager(AccumuloClient client)
     }
 
     @Override
-    public ConnectorSplitSource getSplits(ConnectorTransactionHandle transactionHandle, ConnectorSession session, ConnectorTableHandle tableHandle, SplitSchedulingStrategy splitSchedulingStrategy)
+    public ConnectorSplitSource getSplits(ConnectorTransactionHandle transactionHandle, ConnectorSession session, ConnectorTableHandle tableHandle, SplitSchedulingStrategy splitSchedulingStrategy, DynamicFilter dynamicFilter)
```

Comment:
split the line

Reply:
I wanted to avoid making unrelated formatting changes